Altinn.Platform.Authentication.SystemIntegrationTests/Clients/SystemRegisterClient.cs

```diff
@@ -0,0 +1,38 @@
+using System.Net;
+using Altinn.Platform.Authentication.SystemIntegrationTests.Tests;
+using Altinn.Platform.Authentication.SystemIntegrationTests.Utils;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Altinn.Platform.Authentication.SystemIntegrationTests.Clients;
+
+/// <summary>
+/// For specific requests needed for System Register tests or test data generation purposes
+/// </summary>
+public class SystemRegisterClient : PlatformAuthenticationClient
+{
+    /// <summary>
+    /// For specific Sytem Register Requests
+    /// </summary>
+    public SystemRegisterClient()
```

Comment:
trenger vi en tom constructor?

Reply:
Gjør egentlig ikke det, burde også endre slik at den SystemRegisterClient ikke arver fra PlatformAuthenticationClient tror jeg, for hvis en ende til ende-test bruker flere av de klientene som arver fra den vil den gjøre mye repeterende steg for oppsett osv